recipe/meta.yaml

```diff
@@ -12,19 +12,20 @@ package:
 
 source:
   url: https://github.com/NVIDIA/cccl/archive/refs/tags/v{{ version }}.tar.gz
-  sha256: 7251733c68a05bcb9671dfb0d37e55a5219843d5ba88370234e28bb3c98d545c
+  sha256: 50e9c8bba572a217e7d6748b8c93e76fc42f7c125f6d1da7f117240a14d9a31b
 
 build:
   number: 0
   skip: true  # [osx]
-  skip: true  # [cuda_compiler_version != "11.8"]
```

Comment:
you can also add this back in, then it'll be identical to conda-forge (except for abs.yaml/.gitignore)